pkgs/by-name/op/opensta/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  tcl,
+  swig,
+  bison,
+  flex,
+  git,
+  cudd,
+  eigen,
+  libzip ? null,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "opensta";
+  version = "0-unstable-2025-04-24";
+
+  src = fetchFromGitHub {
+    owner = "parallaxsw";
+    repo = "OpenSTA";
+    rev = "cba0cf5cc43f5ba484bfe8e80f137a6d71dff56e";
+    sha256 = "1v8xicv5syp3x6gn5f2rdn9rdg9pj6x60qdkdik8izzc25s2cjac";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    bison
+    flex
+    git
```

Comment:
I wonder if we could get rid of the Git dependency by supplying the version and / or commit ID externally. `fetchFromGithHub` will by default avoid cloning the repository and instead fetch a tarball, which will not contain the `.git` directory.

Reply:
It seems like the git revision was not being fetched correctly anyways. `sta --version` shows `OpenSTA 2.6.2 GITDIR-NOT`. So I got rid of the git dependency. Or is the rev important to have in the version line?

Reply:
The Git revision might be nice to have to ease (upstream) bug reporting, as we are building unstable versions.